beacon_node/gossipsub/src/types.rs

```diff
@@ -293,6 +300,13 @@ pub struct Prune {
     pub(crate) backoff: Option<u64>,
 }
 
+/// The node requests us to not forward message ids (peer_id + sequence _number) - IDontWant control message.
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct IDontWant {
+    /// A list of known message ids (peer_id + sequence _number) as a string.
+    pub(crate) message_ids: Vec<MessageId>,
```

Comment:
It's a vec here, I guess probably because of the spec right? 

This implementation only ever sends 1 element tho right?